Support-and-Services/saas-releases.md

```diff
@@ -6,6 +6,11 @@ For further information please also visit related pages for Exasol SaaS:
 * [SaaS Documentation](https://docs.exasol.com/saas)
 * [Status Page (status.exasol.com)](https://status.exasol.com)
 
+## March 06
+* make jdbc/oracle drivers uploaded with UI available in DB
+* prevent creation of security rules with invalid CIDR format
+* security improvements
+
```

Comment:
add:
known issues: IPv6 addresses as firewall rules cause issues

Reply:
@andymaclean i think it's a good idea, do you agree?

Reply:
I don't think this works as phrased.  The UI doesn't call this a firewall rule and 'break connections' isn't very specific.  WHat happens?  IPv6 addresses added don't work?  Or IPv6 addresses added break all firewall rules?

If the former I would say "IPv6 addresses do not function correctly in security rules."

If the latter perhaps "Adding IPv6 addresses to the security rules can sometimes prevent other security rules from allowing traffic to access the server.  Remove the IPv6 rule temporarily to fix the issue"

Reply:
I think, once you add an ipv6 rule to your account, any new rule will not apply. also, any new db will not have any of the rules